tools/msvs/find_python.cmd

```diff
@@ -17,6 +17,15 @@ for %%k in ( "HKCU\Software", "HKLM\SOFTWARE", "HKLM\Software\Wow6432Node") do (
   if not errorlevel 1 goto :found-python
 )
 
+:: Check for pyenv-win installation
+set pyenv_path=%USERPROFILE%\.pyenv\pyenv-win\versions
+for /d %%d in ("%pyenv_path%\*") do (
+  if exist "%%d\python.exe" (
+    set p=%%d\
+    goto :found-python
```

Comment:
This will select lowest python version installed via pyenv instead of the selected version. The python bin of the selected version should be determined by `pyenv which python`.